src/DotNet.Status.Web.Tests/AnnotationsControllerTests.cs

```diff
@@ -0,0 +1,104 @@
+using DotNet.Status.Web.Controllers;
+using DotNet.Status.Web.Options;
+using FluentAssertions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.DotNet.Web.Authentication.Tests;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DotNet.Status.Web.Tests
+{
+    [TestFixture]
+    public class AnnotationsControllerTests
+    {
+        private HttpClient _client;
+
+        [SetUp]
+        public void Setup()
+        {
+            var factory = new TestAppFactory();
+
+            factory.ConfigureServices(services =>
+            {
+                services.AddControllers()
+                    .AddApplicationPart(typeof(AnnotationsController).Assembly);
+
+                services.Configure<AnnotationsOptions>(options =>
+                {
+                    options.DeploymentsTableConnectionString = "https://127.0.0.1:10002/devstoreaccount1/deployments";
```

Comment:
Hrm... Can we just mock this out entirely, so we don't need some service running on the local computer to make this work?